mechanics/TOC.md

```diff
@@ -85,6 +84,10 @@ time of the special election). Any replacement TOC member will serve out the
 remainder of the term for the person they are replacing, regardless of the
 length of that remainder.
 
+In the event that a TOC seat becomes available with three months or less left
```

Comment:
bikeshedding: I wonder if we should move this up front (to L73). Then I think we don't have to have the L73 state the more than three months. That would then be else clause?
right now it's kind of like this:
if > 3 months
    - annual election order
 if that didn't work out
    - special election
 if <= 3 months
    - appoint
    
   It might be clearer to just say:
   if <= 3 months
     - appoint
    
    else L73-L85 (minus the mentioning of 3 months, since it was already taken care of above?)

Reply:
ack, updated, ptal